teps/0096-pipelines-v1-release.md

```diff
@@ -0,0 +1,138 @@
+---
+status: proposed
+title: Pipelines V1 Release
```

Comment:
NIT: After reading the proposal, I think the emphasis is on Pipelines V1 API, "Release" might not happen months after V1 API is implemented.

Reply:
good point :)